packages/dev/core/src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -202,6 +209,8 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
         // we don't have to worry about this ray shooting off to infinity. This ray creates
         // a vector defining where we want to zoom to.
         const ray = scene.createPickingRay(scene.pointerX, scene.pointerY, Matrix.Identity(), camera, false);
+        // Since the camera is the origin of the picking ray, we need to offset it by the camera's offset manually
+        ray.origin.subtractInPlace(new Vector3(camera.targetScreenOffset.x, camera.targetScreenOffset.y, 0));

```

Comment:
Could we prevent the new Vector3 for GC here ?

Reply:
Actually, we totally could!  Thanks for catching that!  I'll update it now.